go.mod

```diff
@@ -138,3 +138,5 @@ require (
 	sigs.k8s.io/kube-storage-version-migrator v0.0.6-0.20230721195810-5c8923c5ff96 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 )
+
+replace github.com/openshift/library-go => github.com/machine424/library-go v0.0.0-20240208174128-e2b66c5d004d
```

Comment:
just saw this. Unless this is something that it's a draft or it's depending on a change from a PR to openshift/library-go I'm against this replace here. Mind clarifying this'

Reply:
I assume that that's because of needs https://github.com/openshift/library-go/pull/1676

Reply:
yep https://github.com/openshift/cluster-monitoring-operator/pull/2226#issuecomment-1945681548